### ISSUE TITLE
Running in local

### ISSUE THREAD

**Issue by tayfunyasar** [jmurzy/react-native-foldview#14]
Hi,

I have an error about version.

What is it about?

<img width="465" alt="screen shot 2017-03-30 at 20 11 56" src="https://cloud.githubusercontent.com/assets/1577843/24517195/f48f7fd2-1585-11e7-96da-f0bc136c96c0.png">

**Comment by tayfunyasar** [comment]
closed packager it has disappeared :)

but, what about this? 

<img width="370" alt="screen shot 2017-03-30 at 20 24 23" src="https://cloud.githubusercontent.com/assets/1577843/24517456/116e9966-1587-11e7-8e9c-b99e76c109fa.png">

**Comment by rezof** [comment]
@tayfunyasar quick update on the FoldView.js, make sure to clear cache and reload the app afterwards

```
const baseStyle = this.state.baseLayout ? {
      height: this.state.baseLayout.height,
    } : styles.base;

```
update to
```

const baseStyle = this.state.baseLayout ? {
      height: this.state.baseLayout.height,
      flex: 1
    } : styles.base;

```

**Comment by tayfunyasar** [comment]
Here, 1 more :)

![screen shot 2017-04-03 at 11 22 53](https://cloud.githubusercontent.com/assets/1577843/24601011/ec472e38-185f-11e7-88bb-768dd7721fcd.png)

**Comment by jmurzy** [comment]
I believe this issue is addressed in #13.
